Cargo.toml

```diff
@@ -16,5 +16,16 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 lazy_static = "1.0"
 
+[dev-dependencies]
+criterion = "0.3"
+
+[lib]
+path = "src/lib.rs"
+bench = false  # This option is required for Benchmark CI
```

Comment:
This is copied from icu4x crate using criterion https://github.com/unicode-org/icu4x/blob/a8599e4e3f972b3120887252d290ea6e750cbf60/components/datetime/Cargo.toml#L49-L51

I also notice without defining the lib target, the following lines will be generated when running `cargo bench`, which seems less desired.

```
running 5 tests
test line_breaker::tests::break_rule ... ignored
test line_breaker::tests::linebreak_propery ... ignored
test lstm::tests::thai_word_break ... ignored
test lstm::tests::thai_word_break_utf16 ... ignored
test tests::linebreak ... ignored
```